ui/src/app/styles/flex.tsx

```diff
@@ -0,0 +1,27 @@
+import {reactStyles} from 'app/utils';
```

Comment:
I'm not sure this file is in the right place – is there a reason this doesn't live within app/components/flex.tsx?

These are definitely React components being produced (even though they're lightweight wrappers largely intended to enforce some style invariant). It will probably be more consistent to have this live alongside all other common components, even if it has little to no interactivity.

I'm not wedded to this view, though – I'd be happy to hear a convincing argument in the other direction!

Reply:
Makes sense. Moved it to app/components/flex.tsx